.github/workflows/editor.yaml

```diff
@@ -4,105 +4,66 @@ on:
-          git checkout -b "release/@serlo/editor-$NEXT_VERSION"
-
-      - name: Check if changes occurred after version bump
-        id: check_changes
-        run: |
-          git add .
-          if git diff --exit-code --quiet; then
-            echo "no_changes=true" >> $GITHUB_OUTPUT
-          else
-            echo "no_changes=false" >> $GITHUB_OUTPUT
-          fi
-
-      - name: Commit changes to the new branch
-        if: steps.check_published.outputs.already_published == 'false' && steps.check_changes.outputs.no_changes == 'false'
-        run: |
-          NEXT_VERSION=$(node -p "require('./packages/editor/package.json').version")
-          git add .
-          git commit -m "chore(release): bump @serlo/editor to version $NEXT_VERSION"
-          git push origin "release/@serlo/editor-$NEXT_VERSION"
+        run: yarn editor:publish
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Create Pull Request
-        if: steps.check_published.outputs.already_published == 'false' && steps.check_changes.outputs.no_changes == 'false'
-        id: create_pr
+      - name: Create and push Git tag
+        if: steps.check_published.outputs.already_published == 'false'
         run: |
-          NEXT_VERSION=$(node -p "require('./packages/editor/package.json').version")
-          PR_URL=$(gh pr create --title "Release: Bump @serlo/editor to version $NEXT_VERSION" --body "Automated PR to bump @serlo/editor to $NEXT_VERSION" --base staging --head "release/@serlo/editor-$NEXT_VERSION" --json url -q .url)
-          echo "pr_url=$PR_URL" >> $GITHUB_OUTPUT
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VERSION=$(node -p "require('./packages/editor/package.json').version")
+          git tag v$VERSION
+          git push origin v$VERSION
 
-      - name: Wait for PR to be merged
-        if: steps.create_pr.outputs.pr_url
+      - name: Extract changelog for current version
+        id: extract_changelog
         run: |
-          PR_URL="${{ steps.create_pr.outputs.pr_url }}"
-          while true; do
-            PR_STATE=$(gh pr view $PR_URL --json state -q .state)
-            if [ "$PR_STATE" = "MERGED" ]; then
-              echo "PR has been merged"
-              break
-            elif [ "$PR_STATE" = "CLOSED" ]; then
-              echo "PR was closed without merging"
-              exit 1
-            fi
-            echo "Waiting for PR to be merged..."
-            sleep 60
-          done
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Publish to npm (after PR merge)
-        if: steps.check_published.outputs.already_published == 'false' && steps.create_pr.outputs.pr_url
-        run: yarn editor:publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          CURRENT_VERSION=$(node -p "require('./packages/editor/package.json').version")
```

Comment:
If you stored the CURRENT_VERSION before, you just need to retrieve it from `env.`